beacon-chain/sync/metrics.go

```diff
@@ -15,4 +15,11 @@ var (
 		},
 		[]string{"topic"},
 	)
+	messageReceivedBeforeChainStartCounter = promauto.NewCounterVec(
```

Comment:
what usecases do you think there will be for counting messages before chainstart ?

Reply:
finding spamming nodes

Reply:
spamming nodes ? not sure I understand. It would be normal to get messages from the network if we enter post chainstart

Reply:
+1 not sure if I understand the use case here

Reply:
this counter will count messages recieved before chain start. if we display this counter and it changes during the few months period till genesis. we could take some time to see what are those messages and if some attack is being planned.

Reply:
not super attached to keeping it. thought it might be of an interest